scss/application.scss

```diff
@@ -0,0 +1,3 @@
+.title {
+  color: red;
```

Comment:
Just a comment here. Later we will extract this into another file called `color.scss` and import here.